spec/acceptance/class_spec.rb

```diff
@@ -44,7 +44,7 @@ class { 'powerdns':
     end
 
     describe command('/usr/bin/pdns_control version') do
-      its(:stdout) { is_expected.to match '4.1' }
+      its(:stdout) { is_expected.to match %r{^4\.8} }
```

Comment:
Are you sure this is correct? CI currently does not run the tests

Reply:
See https://github.com/voxpupuli/puppet-powerdns/actions/runs/13873955510/job/38823951320?pr=189 with tests running.